book/src/using-bitfields.md

```diff
@@ -0,0 +1,84 @@
+# Using the Bitfield Types Generated by Bindgen
+
+## Bitfield Strategy Overview
+
+As Rust does not support bitfields, Bindgen generates an opaque struct for each with the following characteristics
```

Comment:
> generates an opaque struct

Nitpick: We don't make the whole struct opaque, we emit an opaque physical *field* that contains one or more logical bitfields.

Reply:
Done